src/cli-ux/styled/table.ts

```diff
@@ -42,7 +42,7 @@ class Table<T extends Record<string, unknown>> {
       extended,
       filter,
       'no-header': options['no-header'] ?? false,
-      'no-truncate': options['no-truncate'] ?? false,
```

Comment:
any reason not to do this in `sf-plugins-core/sfCommand`?

this will affect other oclif CLIs (commands showing sensitive info that was truncated before, CLI scripts parsing tables, etc) and should be a major release.